docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,4 +12,4 @@ _Describe your changes_
 - [ ] All comments are resolved
```

Comment:
Can we have this note here ?
```
Closes  <!--- insert a link to your issues here -->
```
It would be very handy.

There is a nice gh [feature](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

Reply:
Used `resolves` instead